docs/content/en/docs/operator-manual/piped/installation.md

```diff
@@ -93,7 +107,23 @@ Most part is identical to the previous way, but some part is slightly different.
     --set rbac.scope=namespace
   ```
 
-### Installing on single machine
+#### In case on OpenShift less than 4.2
+Containers on OpenShift run using arbitrary Linux users. What we don't know the UID in advance causes a couple of problems.
+To prevent you from such problems, you should set additional options to the previous `helm upgrade` command.
+
+
+``` console
+  --set args.addLoginUserToPasswd=true \
+  --set securityContext.runAsNonRoot=true \
+  --set securityContext.runAsUser=YOUR_UID \
```

Comment:
I feel you, but actually no. Without it, our Piped image tries to use UID 1000 by default. Then it will claim such as:
```
spec.containers[0].securityContext.securityContext.runAsUser: Invalid value: 1000: must be in the ranges: [1000000000, 1000009999]]
```

Reply:
I am curious how the user knows that random UID for configuring to that field. Or they can get that value from the OpenShift admin page?

Reply:
It is. Typically they know their id range (at least our users are aware)

Reply:
I was thinking that they assigned a random ID when the container is started (as their documentation).
Just a thought: if they are only limiting by a range and it seems that range is fixed for all users, we can make `piped-okd` image with a fixed user id in that range and don't have to use `nss_wrapper`.

Reply:
>I was thinking that they assigned a random ID when the container is started (as their documentation). And the user has no way to know what number will be used.

I assumed it works well as long as the UID is within the allocated range.

>Just a thought: if they are only limiting by a range and it seems that range is fixed for all users, we can make piped-okd image with a fixed user id in that range and don't have to use nss_wrapper.

If so good solution! But unfortunately, each users who use OpenShift has their own id range so I would say we can't fix the ID.

Reply:
👍